### PR TITLE
Send method name to Amqp server.

### PR DESCRIPTION
**Pull request by skrater**
Now, the body will be:
`{"method": "doSomething", "args": [0, True, False]}`

### CODE REVIEW

File: client/client.go [hunk 1]
```diff
@@ -35,6 +35,27 @@ type Client struct {
     slotsLock *sync.Mutex
 }
 
+type MessageBody struct {
```
Comment (gfronza) [inline client/client.go]: this type struct could be private, right?

File: client/client.go [hunk 2]
```diff
@@ -35,6 +35,27 @@ type Client struct {
...
+    Args    []interface{}
+}
+
+func (m *MessageBody) Marshal() ([]byte, error){
+    // To not call directly json.Marshal(MessageBody),
+    // maybe some fields in future cannot be exposed, and need specific treatments.
+    type _messageBody struct {
+        Method  string          `json:"method"`
+        Args    []interface{}   `json:"args"`
+    }
```
Comment (gfronza) [inline client/client.go]: IMHO If messageBody was private, it wouldn't be necessary to make the code more complex due to possible future requirements.

File: client/client.go (hunk 1 above)
Comment (skrater) [inline client/client.go]: i think we need this on the server side, to unmarshal messages

**Comment by gfronza** [comment]
@skrater excellent! thank you!!!!

**Pull request merged**